client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -75,7 +75,10 @@ def __init__(self, _host=None, _run_id=None, _from_url=False, **kwargs):
             except KeyError:
                 six.raise_from(EnvironmentError("${} not found in environment".format('VERTA_EMAIL')), None)
             try:
-                self._session.headers.update({_utils._GRPC_PREFIX+'developer_key': os.environ['VERTA_DEV_KEY']})
+                self._session.headers.update({
+                    _utils._GRPC_PREFIX+'developer_key': os.environ['VERTA_DEV_KEY'],
+                    _utils._GRPC_PREFIX+'developer-key': os.environ['VERTA_DEV_KEY'],  # see Client.__init__()
+                })
```

Comment:
Note that this code path in `DeployedModel` only runs if a user instantiates the object directly.
It is _not_ executed when using our `get_deployed_model()` methods, which has been preferred for a while.